Automattic/woocommerce-payments
# Replace svg for credit cards that has a thinner border

Fixes https://github.com/Automattic/woocommerce-payments/issues/1946

#### Changes proposed in this Pull Request
Replaces the credit card icon to have a thinner border.

Before:
![image](https://user-images.githubusercontent.com/572862/121258750-834fcb80-c86c-11eb-9206-c0aa7a1d3f7a.png)

After:
![image](https://user-images.githubusercontent.com/572862/121258785-8ea2f700-c86c-11eb-94a5-9eb62d02d57f.png)

#### Testing instructions
1. Make sure you have both `_wcpay_feature_grouped_settings` and `_wcpay_feature_upe` feature flags enabled
2. Go to `wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments` and make sure the broders of icons are the same as described above.


-------------------
